content/departments/product-engineering/engineering/code-graph/search/guiding_principles.md

```diff
@@ -0,0 +1,41 @@
+# Guiding principles for teamwork on Search Product
+
+This document outlines some guiding principles we seek to follow when welcoming engineers onto the Search Product team. It also serves as shared vocabulary on the team, when we talk about the culture we try to foster.
```

Comment:
I backtracked and see we say "engineers" here after all, dunno if that's intended. If it's just an engineering audience, then please adapt my feedback accordingly :-)

Reply:
So, this is an interesting one. I applied the exact same principles as I was, for example, supporting @benvenker's onboarding. And I believe the state of this document describes how we've grown to / strive to work as a _whole_ team (engineers + EM + PM + design), and a "this only applies to engineers" model would be broken.

That being said, I definitely wrote this from the perspective of "a new engineer is coming onto the team", and I obviously have much more agency in explicitly framing how engineers should approach onboarding onto and operating on the Search Product team. Different expectations may be set by design managers and product directors around how PMs and designers should approach onboarding, interacting, and planning out their time.

@benvenker @marisak @almeidapaulooliveira, curious how you feel about leaving this framed around an engineering audience, vs. a "this is how Search Product operates" framing.

An alternative approach might be to split out general "this is how we _all_ work together" content, from "you're an engineer onboarding onto Search Product, here's what to expect".

Reply:
(merged this so that I can reference the handbook content, but will still welcome opinions on the above comment thread)